### DISCUSSION TITLE
v2版顶栏稍后再看列表不能实时更新

### DISCUSSION THREAD

**Discussion by StqGo** [the1812/Bilibili-Evolved#2707] [提问 / Q&A] [answered]
当我取消一个视频的 稍后再看 状态后，顶栏的 稍后再看 列表并不会立即更新，需要手动刷新一下
v1是可以立即更新的 昨天更新到v2发现这个问题后 试了很多次都不行
不知道是不是v2的bug  还是说暂时还没有做这个呢？

**Comment by the1812** [comment, marked as answer]
v2.0.8 已经实现了

**Reply by StqGo** [reply]
收到收到  感谢大佬！ 前两天已经解决啦
